documentation/src/main/asciidoc/topics/upgrading.adoc

```diff
@@ -1,3 +1,17 @@
+= Upgrading from 10.1 to 11.0
+
+== Maximum Idle Timeouts with Clustered Cache Modes
+Infinispan maximum idle expiration has been changed to improve performance with
```

Comment:
It actually makes performance worse :)

It improves consistency under node failure.